firmware/rhizome.ino

```diff
@@ -7,8 +7,9 @@
 #include "Ohmbrewer_Publisher.h"
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_RIMS.h"
+#include "Ohmbrewer_Onewire.h"
 #include "onewire.h"
-// #include "Ohmbrewer_Publisher.h" // (*)
+#include <list>
```

Comment:
You should include list down below the undef's where we include deque. Those undef's guard us against some Particle-specific shenanigans.